google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreImpl.java

```diff
@@ -135,26 +143,57 @@ void setPrevTransactionId(ByteString transactionId) {
       options = options.toBuilder().setReadWrite(readWrite).build();
     }
 
+    private io.opentelemetry.api.trace.Span startSpanInternal(
```

Comment:
Nit: The functionality is a bit unclear to me from the name `startSpanInternal` itself. Could we add some comments or rename this function, e.g.`startSpanWithParentContext`?

Reply:
Good point. Renamed the function.